images/image.mk

```diff
@@ -15,6 +15,8 @@
 # remove default suffixes as we dont use them
 .SUFFIXES:
 
+override GOOS=linux
```

Comment:
what effect does this have when building on say darwin?

Reply:
yes! building Darwin images is not supported.